test/runtests.jl

```diff
@@ -4,5 +4,5 @@ using Test, TestSetExtensions
 using WrappedMultiGraphs
 
 @testset "AttributeGraphs.jl" begin
-    @includetests ["constructors","graphs", "attributes"]
+    @includetests ["constructors","graphs", "attributes",  "opinionatedtest"]
```

Comment:
"opinionatedtest" follows a different naming convention. in case a new file is needed I would prefer it being called just "opinionated" to match the other files in the `./test` directory